crates/blockchain/payload.rs

```diff
@@ -547,6 +547,7 @@ impl Blockchain {
         context.payload.header.requests_hash = context.requests_hash;
         context.payload.header.gas_used = context.payload.header.gas_limit - context.remaining_gas;
         context.account_updates = account_updates;
+
```

Comment:
Try not to include pretifier changes in such a big PR. This makes review more complicated.